### ISSUE TITLE
Redo date calculations with IntlDateTimeFormatter

### ISSUE THREAD

**Issue by meezaan** [islamic-network/api.aladhan.com#29]
Like this example:

```
$DateTime = new DateTime();
$IntlDateFormatter = new IntlDateFormatter(
                    'en_US@calendar=islamic-civil', // or islamic
                    IntlDateFormatter::FULL,
                    IntlDateFormatter::FULL,
                    'Asia/Tehran',
                    IntlDateFormatter::TRADITIONAL);

echo $IntlDateFormatter->format($DateTime);
```

**Comment by papasmile** [comment]
Salam, @meezaan couple questions:
1) Are you talking about timings/v1/prayerTimes.php, timings/v1/dateAndTime.php, or other routes as well?
2) Are you thinking to change the * output * of any of those routes, or just internal calculations?

**Comment by meezaan** [comment]
No, this refers to https://github.com/islamic-network/api.aladhan.com/blob/master/src/AlAdhanApi/Model/HijriCalendarService.php in general. There are always discrepancies with Hijri dates.

The ```IntlDateFormatter```seemed a better approach but it also has discrepancies, so I'm in 2 minds about whether to do this or not. I've started work on a library try and reconcile the differences, but have not got very far.

**Comment by meezaan** [comment]
Calendar calculations have been changed to use https://github.com/islamic-network/calendar.
